packages/bridge-status-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [15.0.0]
+
+### Changed
+
+- **BREAKING:** Bump `@metamask/assets-controllers` peer dependency to `^59.0.0` ([#5712](https://github.com/MetaMask/core/pull/5712))
```

Comment:
```suggestion
- **BREAKING:** Bump `@metamask/assets-controllers` peer dependency to `^59.0.0` ([#5712](https://github.com/MetaMask/core/pull/5712))
- **BREAKING:** Bump `@metamask/bridge-status-controllers` peer dependency to `^15.0.0` ([#5712](https://github.com/MetaMask/core/pull/5712))
```

Reply:
Indeed I missed this one before approving

Reply:
Indeed, wrong controller 😁 thanks for applying the change!